include/knowhere/config.h

```diff
@@ -473,6 +473,11 @@ class BaseConfig : public Config {
         }
         return omp_get_max_threads();
     }
+
+    virtual Status
+    Customize(const Json& json) {
```

Comment:
This function name express no useful info about what it does. Let's rename it to `IsValid` or `CheckValidation` something like this

Reply:
this API is not a readonly API, it may modify the value in the Config.

Reply:
`CheckAndAdjustConfig` sounds better to me then, since we need to let people know what is this API doing.

Reply:
rename API to `CheckAndAdjustConfig`